### ISSUE TITLE
Alerts don't stay on page (make autohide optional)

### ISSUE THREAD

**Issue by FalonMadeIt** [themeteorchef/bert#23]
Please make it so the alerts can remain on the page until the user makes another action or clicks another button. Making autohide optional will make persisting alerts. Users will be able to look away from the screen without missing anything or change a template without an old alert lingering.

Thanks

**Comment by alur222** [comment]
@falont23 did u manage to make this work? I also need this feature.

**Comment by FalonMadeIt** [comment]
No, I made a workaround.  I think I set autohide so messages would remain for a really long time. Also I refreshed the session when the templates changed.
